internal/pkg/cli/svc_init.go

```diff
@@ -389,6 +407,38 @@ func (o *initSvcOpts) askSvcName() error {
 	return nil
 }
 
+func (o *initSvcOpts) askIngressType() error {
+	if o.wkldType != manifest.RequestDrivenWebServiceType || o.ingressType != "" {
+		return nil
+	}
+
+	opts := []prompt.Option{
+		{
+			Value: ingressTypeEnvironment,
+		},
+		{
+			Value: ingressTypeInternet,
+		},
+	}
+
+	t, err := o.prompt.SelectOption(svcInitIngressTypePrompt, svcInitIngressTypeHelpPrompt, opts, prompt.WithFinalMessage("Reachable from:"))
+	if err != nil {
+		return fmt.Errorf("select ingress type: %w", err)
+	}
+	o.ingressType = t
+	return nil
+}
+
+func (o *initSvcOpts) validateIngressType() error {
+	if o.wkldType != manifest.RequestDrivenWebServiceType {
+		return nil
+	}
+	if strings.EqualFold(o.ingressType, "internet") || strings.EqualFold(o.ingressType, "environment") {
+		return nil
+	}
+	return fmt.Errorf("invalid ingress type %q", o.ingressType)
```

Comment:
Can we list the valid options in the error message to help users figure out what should be provided?